lib/pandadoc/api/document.rb

```diff
@@ -112,12 +100,7 @@ def link(token, document_id, params = {})
       end
 
       def download(token, document_id)
-        HTTParty.get("#{Pandadoc::Api::API_ROOT}/documents/#{document_id}/download",
-                     headers: { 'Authorization' => auth_header(token) })
-      end
-
-      def auth_header(token)
-        "Bearer #{token}"
+        client.get "/documents/#{document_id}/download", token
       end
 
       def validated_params(params, validations)
```

Comment:
This can probably also be broken out as it's the same across both files.

Reply:
I was thinking the same thing. Going to open a separate PR to capture that.